app/modules/site_settings/models.py

```diff
@@ -86,8 +86,8 @@ class SiteSetting(db.Model, Timestamp):
         'site.general.donationButtonUrl': {
             'type': str,
             'definition': {
-                'displayType': 'longstring',
-                'fieldType': 'string',  # TODO so why is this not url?
+                'displayType': 'string',
+                'fieldType': 'url',  # TODO so why is this not url?
```

Comment:
probably drop the comments?  or maybe there is a context there i am not understanding.

Reply:
Good call, that would have annoyed me too